vulnerabilities/templates/base.html

```diff
@@ -6,6 +6,7 @@
 <link rel="stylesheet" href="{% static 'css/bulma.css' %}"/>
 <link rel="stylesheet" href="{% static 'css/custom.css' %}"/>
 <link rel="stylesheet" href="{% static 'css/font-awesome.css' %}"/>
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
```

Comment:
We don't want to use bootstrap. We already use bulma. See https://bulma.io/documentation/elements/table/